web/web.go

```diff
@@ -26,13 +26,14 @@ import (
 
 	pprof_runtime "runtime/pprof"
 
+	clientmodel "github.com/prometheus/client_golang/model"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/log"
-	"github.com/prometheus/prometheus/util/route"
 
-	clientmodel "github.com/prometheus/client_golang/model"
+	api_legacy "github.com/prometheus/prometheus/web/api/legacy"
+	api_v1 "github.com/prometheus/prometheus/web/api/v1"
```

Comment:
See above. Perhaps just `legacy` and `v1` works well here because it tranlates to `legacy.API` and `v1.API` below.